packages/api/src/clickhouse/index.ts

```diff
@@ -1090,9 +1089,36 @@ export const getSessions = async ({
     ],
   );
 
+  const sessionsWithRecordingsQuery = SqlString.format(
+    `WITH sessions AS (${sessionsWithSearchQuery}),
```

Comment:
nit: would be nice to merge queries for readablility (also make query parameterization migration a bit easier). but no big deal, I can clean it up later

Reply:
In this case this query execution is conditional (as this is a slower path), unless you were thinking of having the original query CTE duplicated?

Reply:
yeah, I was thinking to have CTE inline basically

Reply:
I'ma merge this as-is right now, and we can probably revisit that later - likely we'll need to tune this due to perf anyways.